yt_dlp/postprocessor/embedthumbnail.py

```diff
@@ -90,12 +90,31 @@ def run(self, info):
 
         success = True
         if info['ext'] == 'mp3':
-            options = [
-                '-c', 'copy', '-map', '0:0', '-map', '1:0', '-write_id3v1', '1', '-id3v2_version', '3',
-                '-metadata:s:v', 'title="Album cover"', '-metadata:s:v', 'comment=Cover (front)']
+            # Method 1: Use mutagen
+            # Prioritize mutagen over ffmpeg since ffmpeg messes up the lyrics data
+            if mutagen:
+                try:
+                    self._report_run('mutagen', filename)
+                    audio = mutagen.id3.ID3(filename)
+                    if 'APIC' in audio:
+                        del audio['APIC']
+                    with open(thumbnail_filename, 'rb') as thumbfile:
+                        audio['APIC'] = mutagen.id3.APIC(
+                            encoding=mutagen.id3.Encoding.UTF8, mime='image/%s' % thumbnail_ext, type=3,
+                            desc=u'Cover (front)', data=thumbfile.read())
+                    audio.save()
+                    temp_filename = filename  # Mutagen saves to the original file
+                except Exception as err:
+                    self.report_warning(f'unable to embed using mutagen; {err}')
+                    success = False
+            # Method 2: Use ffmpeg
+            else:
```

Comment:
```suggestion

            # Method 2: Use ffmpeg
            if not success:
```

This way ffmpeg is used in case mutagen errors